Justfile

```diff
@@ -2,8 +2,16 @@
 default:
   just --list
 
-# Run all checks from CI
-ci: format test
+# Run all pull request checks from CI via act
+ci:
+  act --container-architecture=linux/amd64 pull_request
+
+# Run Neovim nightly tests from CI via act
+ci-nightly:
```

Comment:
<img width="1381" alt="image" src="https://github.com/johmsalas/text-case.nvim/assets/6575405/cdadc1c8-0636-4f4f-8e7e-72fe7f1fe2b4">

Reply:
That's really strange. For me it looks like this:

<img width="843" alt="Screenshot 2023-11-18 at 06 47 32" src="https://github.com/johmsalas/text-case.nvim/assets/4454052/6f75b1b8-a2e0-43d8-ac7c-c7c0ef421a5a">

My act version is 0.2.54

Reply:
Does it work for you when you specify the workflow file?

```
act --container-architecture=linux/amd64 -j TestsOnNightly -W .github/workflows/neovim-nightly.yml
```

Reply:
Was an issue in my local setup, git wasn't pulling the branch, therefore the file did not exist